text/0000-security-advisories.md

```diff
@@ -0,0 +1,259 @@
+- Feature Name: security_advisories
+- Start Date: 2016-08-24
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Crates.io should offer an API to release security advisories for crates. Cargo
+should offer a new command for the same purpose and warn about vulnerable crate
+versions during compilation.
+
+# Motivation
+[motivation]: #motivation
+
+When compared to other ecosystems such as Python's, Rust's broader community
+prefers many single-purpose crates over larger monoliths. This situation,
+together with the strongly encouraged practice of pinning MINOR versions of
+dependencies, slows down the propagation of critical security fixes.
```

Comment:
I'm not aware of pinning minor versions in `Cargo.toml` being encouraged (except the `0.x.y` case but then `x` is the major version for cargo-semver purposes). Can you elaborate on this?